isaric/parsers/ccp-netherlands.toml

```diff
@@ -0,0 +1,736 @@
+  ref = "Y/N"
+
+[subject.has_smoking]
+  field = "Smoking"
+
+  [subject.has_smoking.values] # these 3 are present, who knows if the mapping is correct
+    1 = "yes"
+    2 = "former"
+    3 = "never"
+
+[subject.has_asthma]
+  field = "asthma"
+  ref = "Y/N"
+
+[subject.has_diabetes]
+  combinedType = "any"
+  fields = [
+    { field = "diabetes_complications", ref = "Y/N" },
+    { field = "diabetes_without_complications", ref = "Y/N" },
+  ]
+
+[subject.has_liver_disease]
+  combinedType = "any"
+  fields = [
+    { field = "live_disease", ref = "Y/N" },
+    { field = "mld", ref = "Y/N" },
+  ]
+
+[subject.has_malnutrition]
+  field = "Malnutrition"
+  ref = "Y/N"
+
+[subject.has_comorbidity_other]
+  combinedType = "set"
+  excludeWhen = "none"
+  fields = [{ field = "Specif_other_risk_factors" }]
+
+[subject.has_died]
+  field = "Outcome"
+  values = { 4 = true }
+
+[subject.date_death]
+  field = "Outcome_dt"
+  if = { Outcome = 4 }
+
+[subject.icu_admitted]
+  field = "ICU_High_Dependency_admission"
+  ref = "Y/N"
+
+  # VISIT
+  # required fields - "visit_id", "subject_id", "country_iso3", "start_date", "outcome", "date_outcome"
+
+[visit]
+  country_iso3 = "NLD"
+
+  [visit.visit_id]
+    field = "record_id"
+    sensitive = true
+
+  [visit.subject_id]
+    field = "record_id"
+    sensitive = true
+
+  [visit.start_date]
+    field = "admission_dt"
+
+  [visit.pathogen_test_date]
+    field = "collection_dt"
+    if = { Pathogen_tested_Detected_result = { "=~" = "sars-cov-2" } }
+
+  [visit.icu_admission]
+    combinedType = "any"
+    fields = [
+      { field = "current_in_ICU_ITU_IMU_HDU", ref = "Y/N" },    # these are over different lines, so probably won't work
+      { field = "ICU_High_Dependency_admission", ref = "Y/N" },
+    ]
+
+  [visit.icu_admission_dates]
+    combinedType = "set"
+    excludeWhen = "none"
+    fields = [{ field = "Admission_dt_icu" }]
+
+  [visit.transfer_from_other_facility]
+    field = "facility_transfer"
+    values = { 1 = true, 2 = true, 3 = false }
+
+  [visit.treatment_dialysis]
+    combinedType = "any"
+    fields = [
+      { field = "patient_interventions#DialysisHemofiltration", values = { 1 = true, 0 = false } },
+      { field = "RRT_dialysis", ref = "Y/N" },
+    ]
+
+  [visit.treatment_inotropes_vasopressors]
+    combinedType = "any"
+    fields = [
+      { field = "patient_interventions#Any_vasopressorinotropic_support", values = { 1 = true, 0 = false } },
+      { field = "Inotropes_vasopressors", ref = "Y/N" },
+    ]
+
+  [visit.treatment_ecmo]
+    combinedType = "any"
+    fields = [
+      { field = "patient_interventions#Extra_corporeal_life_support_ECLS", values = { 1 = true, 0 = false } },
+      { field = "Extracorporeal_support", ref = "Y/N" },
+    ]
+
+  [visit.treatment_corticosteroid]
+    field = "Corticosteroid"
+    ref = "Y/N"
+
+  [visit.treatment_corticosteroid_type]
+    field = "Corticosteroid_type_dose"
+    apply.function = "wordSubstituteSet"
```

Comment:
Would we generally need this function if we used combinedType = "set"?
assuming the values of the field are text?

Reply:
No, this just allows us to search for words/phrases in free-text and replace them with e.g. English spelling, or a standardised capitalisation format.

Reply:
gotcha!